presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaQueryRunnerBuilder.java

```diff
@@ -37,7 +37,7 @@
     protected final T testingKafka;
```

Comment:
I can extract single commit per change if you like. I thought that having 5 commits per simple change is an overkill, on the other hand this commit is not super readable either. Please let me know what do you think about it.